nservicebus/rawmessaging/index_content_core_[8,).partial.md

```diff
@@ -0,0 +1,29 @@
+Since version 8 of NServiceBus the transport infrastructure can be used directly without the need to spin up full NServiceBus endpoint. This is especially useful when dealing with integrations with 3rd party systems and building message gateways or bridges.
```

Comment:
```suggestion
Since version 8 of NServiceBus, the transport infrastructure can be used directly without the need to spin up a full NServiceBus endpoint. This is especially useful when dealing with integrations with 3rd party systems and building message gateways or bridges.
```